tools/NuGet/BuildDynamoCore.bat

```diff
@@ -16,12 +16,6 @@ mkdir .\%core%\tools
 copy %base%\DynamoCore.dll .\%core%\lib\%framework%\DynamoCore.dll
 copy %base%\DynamoCore.xml .\%core%\build\%framework%\DynamoCore.xml
 
-copy %base%\ProtoGeometry.dll .\%core%\lib\%framework%\ProtoGeometry.dll
```

Comment:
Can you please make sure that we build from Release folder rather than the Debug folder? Please update the %base% to point to release folder.